db/schema.rb

```diff
@@ -139,7 +139,6 @@
     t.float    "max_value"
     t.float    "min_value"
     t.datetime "calibrated_at"
-    t.boolean  "smaxtec_sensor"
```

Comment:
@tillprochaska that's a good find @matthib the `schema.rb` was generated. So I believe this is a leftover of the schema.rb that was merge into master? Do we need to care about `smaxtec_sensor`?